lib/model/Product.dart

```diff
@@ -231,9 +231,17 @@ class Product extends JsonObject {
       toJson: IngredientsAnalysisTags.toJson)
   IngredientsAnalysisTags? ingredientsAnalysisTags;
 
-  @JsonKey(
-      name: 'nutriments', includeIfNull: false, toJson: Nutriments.toJsonHelper)
-  Nutriments? nutriments;
+  /// When no nutrition data is true, nutriments are always null
+  /// This logic is handled by the getters/setters of [noNutritionData] and
+  /// [nutriments]
+  @JsonKey(ignore: true)
```

Comment:
We don't need `@JsonKey` there, do we?

Reply:
Json Serialize doesn't work with private fields